commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/http4/test/Http4GetContentInfoTest.java

```diff
@@ -56,4 +59,13 @@ public void testGetContentInfo() throws FileSystemException {
         // Used to NPE before fix:
         content.getContentInfo();
     }
+
+
+    @Test
+    public void testGetZeroContents() throws IOException {
+        final FileSystemManager fsManager = VFS.getManager();
+        final FileObject fo = fsManager.resolveFile("http4://monpremierlivre.home.blog/");
```

Comment:
I think you will want to use Mockito here instead of depending on a random third party web site -- random from Apache's POV ;-)

Reply:
I'd rather see a local server here because this test requires outbound access and is brittle.